.github/fixtures/test-fixtures-locally.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
This file is for running fixtures tests like the following:

```sh-session
~/ghq/github.com/orhun/git-cliff use-root-commit-when-latest-and-one-tag
❯ ./.github/fixtures/test-fixtures-locally.sh test-ignore-tags
Initialized empty Git repository in /tmp/test/.git/
[main (root-commit) 074b290] feat: add skip feature
[main 7d0f941] feat: add feature 1
[main 33d4d94] feat: fix feature 1
[main 8e42066] feat: add feature 2
[main cb78cbe] feat: add feature 3

---Run git-cliff---
   Compiling git-cliff-core v0.5.0 (/home/kenji/ghq/github.com/orhun/git-cliff/git-cliff-core)
   Compiling git-cliff v0.5.0 (/home/kenji/ghq/github.com/orhun/git-cliff/git-cliff)
    Finished dev [unoptimized + debuginfo] target(s) in 3.56s
     Running `/home/kenji/ghq/github.com/orhun/git-cliff/target/debug/git-cliff --config /home/kenji/ghq/github.com/orhun/git-cliff/.github/fixtures/test-ignore-tags/cliff.toml`
# Changelog

All notable changes to this project will be documented in this file.

## [0.2.0] - 2021-01-22

### Feat

- Add feature 2
- Add feature 3

## [0.1.0] - 2021-01-22

### Feat

- Add feature 1
- Fix feature 1

<!-- generated by git-cliff -->
```

Reply:
It's not necessary for this PR, but I think sometimes it would be useful when testing.